bazel/io_opentracing_cpp.patch

```diff
@@ -57,3 +57,76 @@ index 1721fb3..3873b3a 100644
  endif()
  
  # ==============================================================================
+diff --git a/include/opentracing/config.h b/include/opentracing/config.h
```

Comment:
These were copied from the output of envoys previous build to ensure they match (installing a host CMake and not executing in Docker for "bootstrapping", but this won't be required for using them now that they are here)